src/wizard/javascript/angular.md

```diff
@@ -4,148 +4,244 @@ doc_link: https://docs.sentry.io/platforms/javascript/angular/
 support_level: production
 type: framework
 ---
-On its own, `@sentry/browser` will report any uncaught exceptions triggered from your application. Additionally, you can configure `@sentry/browser` to catch any Angular-specific exceptions reported through the [@angular/core/ErrorHandler](https://angular.io/api/core/ErrorHandler) component. This is also a great opportunity to collect user feedback by using `Sentry.showReportDialog`.
 
-First, install `@sentry/browser`:
+To use Sentry with your Angular application, you will need to use `@sentry/angular` (Sentry’s Browser Angular SDK).
+
+<Alert level="info" title="Note"><markdown>
+
+`@sentry/angular` is a wrapper around the `@sentry/browser` package, with added functionality related to Angular. All methods available in the `@sentry/browser` package can also be imported from `@sentry/angular`.
+
+</markdown></Alert>
+
+Add the Sentry SDK as a dependency using yarn or npm:
 
 ```bash
 # Using yarn
-yarn add @sentry/browser
+$ yarn add @sentry/angular
 
 # Using npm
-npm install @sentry/browser
+$ npm install @sentry/angular
 ```
 
-Then initialize a new Sentry instance and configure Angular with the `ErrorHandler` provided and explained below:
+### Connecting the SDK to Sentry
 
-```typescript
-import { BrowserModule } from "@angular/platform-browser";
-import { NgModule, ErrorHandler, Injectable } from "@angular/core";
-import { HttpErrorResponse } from "@angular/common/http";
+After you've completed setting up a project in Sentry, Sentry will give you a value which we call a _DSN_ or _Data Source Name_. It looks a lot like a standard URL, but it’s just a representation of the configuration required by the Sentry SDKs. It consists of a few pieces, including the protocol, public key, the server address, and the project identifier.
 
-import { environment } from "../environments/environment";
-import { AppComponent } from "./app.component";
+You should `init` the Sentry browser SDK as soon as possible during your application load up, before initializing Angular:
 
-import * as Sentry from "@sentry/browser";
+```javascript
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { init } from '@sentry/angular';
```

Comment:
Is there some smart idea behind departing from the `import * as Sentry from` form?

I thought `Sentry.init` was clearer than `init` in intent and eventually people copy pasting the code will also want to do something like `Sentry.captureMessage`, etc.